src/aiortc/contrib/getkeypoints.py

```diff
@@ -15,16 +14,19 @@
 class KeypointsGenerator():
     def __init__(self):        
         self.to_tensor = transforms.ToTensor()
-        self.fa = face_alignment.FaceAlignment(face_alignment.LandmarksType._2D, flip_input=True, device='cpu')
+        self.fa = face_alignment.FaceAlignment(face_alignment.LandmarksType._2D, \
+                                               flip_input=True, device='cpu')
 
     def get_keypoints(self, input_frames, image_size = 256, crop_data = False):
-        """Generates dataset images, keypoints (also called poses), and segmenatations from input_frames/ frames
+        """Generates dataset images, keypoints (also called poses),
+         and segmenatations from input_frames/ frames
 
         Inputs
```

Comment:
would simplify this commenting structure as I mentioned to not have separate "inputs" because of the style of this codebase. Would rather work it into the line comment instead

Reply:
same thing with returns

Reply:
Yeah so this is just following up on the old review comment I already had

Reply:
removed the unnecessary comments